docs/Demo/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from '@deque/cauldron-react/';
+
+const Footer = props => {
+  const year = new Date().getFullYear();
+
+  return (
+    <footer className="Footer">
+      <ul className="Footer-links">
```

Comment:
If we are going with BEM, since `links` is an element


```suggestion
      <ul className="Footer__links">
```